### PR TITLE
Added ARF reference in PID claim

### LINKED ISSUE #193: VC (PID / (Q)EAA) data model update
Add a class to categorise data within a credential, distinguishing for instance between "dati personali" (including current_first_name, current_family_name, date_of_birth, place_of_birth, taxpayer_identification_number, ...) and "dati tessera". This will allow the information to be organised in a more readable way at the front-end.

### LINKED ISSUE #261: [PID/(Q)EAA Data Model - PID Claims] missing element
Within PID claims, [Link ](https://github.com/italia/eudi-wallet-it-docs/blob/df948503e22cbf949530a07b6cee0e83e6901613/docs/en/pid-eaa-data-model.rst?plain=1#L109) we need to add a reference to the Schema used by the PID.

### LINKED ISSUE #262: [PID/(Q)EAA Data Model - PID Claims] Family name
PID Claim given_name AND family_name as defined would bring possible limitations:

1) this requirement could bring to issues when applied to situations where a user may have multiple names/surnames (e.g. see latest Italian legislation on newborns from 2022) and in case of patronimic
2) there's no provision on what would be the corresponding structure for a legal-entity identifier

### PR DESCRIPTION
**Pull request by cmarco0**
this commit aims to resolve the following issues:

- https://github.com/italia/eudi-wallet-it-docs/issues/261
- https://github.com/italia/eudi-wallet-it-docs/issues/262

### CODE REVIEW

**Comment by peppelinux** [comment]
This PR doesn't resolve the mentioned issues, it only adds a reference to the ARF

**Comment by cmarco0** [comment]
> This PR doesn't resolve the mentioned issues, it only adds a reference to the ARF

Yes, I have added the references to the ARF where the PID data set schema is provided (PID Rulebook).

Then, in the table here [PID Claims](https://github.com/italia/eudi-wallet-it-docs/blob/versione-corrente/docs/en/pid-eaa-data-model.rst#pid-claims), the reference for family_name state as per [OpenID Connect Core 1.0 Standard Claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims):
Surname(s) or last name(s) of the End-User. Note that in some cultures, people can have multiple family names or no family name; all can be present, with the names being separated by space characters.

Additionally, in the PID Rulebook table [PID Rulebook](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-3/annex-3.01-pid-rulebook.md#232-overview), the Encoding format is tstr, referenced in [RFC 8610](https://www.rfc-editor.org/rfc/rfc8610.html). This format can accommodate multiple family names separated by space characters, as explained in the OpenID specification.

**Review by peppelinux — APPROVED** [review]

File: docs/en/pid-eaa-data-model.rst [hunk 1]
```diff
@@ -134,6 +134,8 @@ Depending on the Digital Credential type **vct**, additional claims data MAY be
       - [SD]. National tax identification code of natural person as a String format. It MUST be set according to ETSI EN 319 412-1. For example ``TINIT-<ItalianTaxIdentificationNumber>``
       - 
 
+The the PID attribute schema is defined by `ARF v1.4 <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/arf.md#21-identification-and-authentication-to-access-online-services>`_, as detailed in the `PID Rulebook <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-3/annex-3.01-pid-rulebook.md#23-pid-attributes>`_.
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
The PID attribute schema, which encompasses all potential user data, is defined in `ARF v1.4 <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/arf.md#21-identification-and-authentication-to-access-online-services>`_, and furthermore detailed in the `PID Rulebook <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-3/annex-3.01-pid-rulebook.md#23-pid-attributes>`_.
```

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: docs/en/pid-eaa-data-model.rst [hunk 2]
```diff
@@ -134,6 +134,8 @@ Depending on the Digital Credential type **vct**, additional claims data MAY be
       - [SD]. National tax identification code of natural person as a String format. It MUST be set according to ETSI EN 319 412-1. For example ``TINIT-<ItalianTaxIdentificationNumber>``
       - 
 
+The PID attribute schema, which encompasses all potential user data, is defined in `ARF v1.4 <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/arf.md#21-identification-and-authentication-to-access-online-services>`_, and furthermore detailed in the `PID Rulebook <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-3/annex-3.01-pid-rulebook.md#23-pid-attributes>`_.
```
Comment (peppelinux) [inline docs/en/pid-eaa-data-model.rst]:
```suggestion
The PID attribute schema, which encompasses all potential User data, is defined in `ARF v1.4 <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/arf.md#21-identification-and-authentication-to-access-online-services>`_, and furthermore detailed in the `PID Rulebook <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-3/annex-3.01-pid-rulebook.md#23-pid-attributes>`_.
```

**Review by peppelinux — APPROVED** [review]

**Review by grausof — APPROVED** [review]

**Review by fmarino-ipzs — APPROVED** [review]

**Pull request merged**